nengo/spa/tests/test_basalganglia.py

```diff
@@ -22,6 +22,7 @@ def test_basal_ganglia(Simulator, seed, plt):
             'dot(vision,CAT)+0.5-dot(vision,CAT) --> motor=E',
             'dot(vision,PARROT) + compare --> motor=F',
             '0.5*dot(vision,MOUSE) + 0.5*compare --> motor=G',
+            '( dot(vision,MOUSE) - compare) * 0.5 --> motor=H'
```

Comment:
Not sure, if I agree ... and even if shouldn't be there a space in front of the closing parenthesis for symmetry reasons too?

And given that are discussing legibility: Why are there no spaces after commas and around operators? (I also don't like to write semantic pointers in all uppercase, it's so 90ies HTML like. But I don't have hope to get this one changed.)